internal/auth/subtype.go

```diff
@@ -58,6 +58,18 @@ var (
 	_ Account = (*password.Account)(nil)
 )
 
+type ManagedGroup interface {
```

Comment:
where/how is this used?  I couldn't find any current references.  Perhaps, it's need in future managed groups PRs?

Reply:
Yep, it's used later in the service handler (PR to come).